csharp/test/Slice/alias/Client.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) ZeroC, Inc. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Test;
+
+namespace ZeroC.Slice.Test.Alias
```

Comment:
I think it will be better if we have a test that actually exercises these aliases, checking that the operation has the expected return type doesn't ensure they work, something might be wrong with the marshaling or unmarshaling when using type alias and we will not notice it.

Reply:
Fair enough, I'll add more to the test for actually using them.